### DISCUSSION TITLE
Unhandled 'error' event at startup of the Quire server

### DISCUSSION THREAD

**Discussion by yl5682** [thegetty/quire#601] [Get Help]
I had a fresh Quire installation on macOS 12.6. Specifically my environment is configured as:

- node.js v14.16.0
- npm 8.19.2
- quire v0.20.4

My challenge is that, right after the installation and creation of a new Quire project, the `quire preview` does not run. It will give me an error as such. I have tried reinstalling the Node/Quire but it didn't work (after removing the historic versions).

<img width="788" alt="Screen Shot 2022-11-29 at 3 36 30 AM" src="https://user-images.githubusercontent.com/7308403/204544203-4c77521b-2d77-4c79-99b7-00107a7c3609.png">

Stuck here. Would really appreciate any input/pointers. 

Thank you!

**Comment by Erin-Cecele** [comment]
Hi @yl5682. Try running `quire install` while in your new project directory and then run `quire preview` again. Let us know if that resolves the issue. You'll also want to make sure you are working in this version of Node: https://nodejs.org/download/release/v14.18.1/node-v14.18.1.pkg.

**Reply by Erin-Cecele** [reply]
Hi @yl5682. Were you able to get this issue resolved?
